### PR TITLE
Autoplace integration

### PR DESCRIPTION
**Pull request by Glease**
Depends on https://github.com/GTNewHorizons/GT5-Unofficial/pull/1245

### CODE REVIEW

File: src/main/java/gtPlusPlus/preloader/asm/transformers/ClassTransformer_GT_Client.java [hunk 1]
```diff
@@ -81,6 +82,10 @@ public ClassTransformer_GT_Client(byte[] basicClass, boolean obfuscated) {
 		FMLRelaunchLog.log("[GT++ ASM] Gregtech Client Proxy Patch", Level.INFO, "Fixing Issues in vanilla GT.");		
 		if (mModern) {
 			aTempReader.accept(new MethodAdaptor2(aTempWriter), 0);
+			if (mGTNH) {
```
Comment (draknyte1) [inline src/main/java/gtPlusPlus/preloader/asm/transformers/ClassTransformer_GT_Client.java]: I don’t think there’s any point doing this anymore as it’s certainly only compatible with GTNH after the last 6mo of reworks. 🤔

Comment (Glease) [reply src/main/java/gtPlusPlus/preloader/asm/transformers/ClassTransformer_GT_Client.java]: this poor incomplete change isn't even meant to made it into the PR. it belongs to another project that attempt to extract gt++ asm to another mod. thanks for pointing out.

Comment (Glease) [reply src/main/java/gtPlusPlus/preloader/asm/transformers/ClassTransformer_GT_Client.java]: In all regards it feels bad to fork a oss project and not able to contribute back.. I'll still attempt to reduce the amount of breaking changes done, but will not hesitate to do so if the situation requires it.

**Review by Dream-Master — APPROVED** [review]
live on zeta

**Pull request merged**